### PR TITLE
Generated tapi

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
Generated Tapi models.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ import com.adyen.Client;
 import com.adyen.enums.Environment;
 import com.adyen.service.TerminalCloudAPI;
 import com.adyen.model.nexo.*;
-import com.adyen.model.terminal.*;
+import com.adyen.model.terminalOld.*;
```
Comment (wboereboom) [inline README.md:288]: what is terminalOld?

File: src/main/java/com/adyen/model/terminal/AmountsResp.java [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+   * Get authorizedAmount
+   * minimum: 0.0
+   * maximum: 99999999.999999
+   * @return authorizedAmount
+  **/
+  @ApiModelProperty(required = true, value = "")
+  @JsonProperty(JSON_PROPERTY_AUTHORIZED_AMOUNT)
+  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+
+  public BigDecimal getAuthorizedAmount() {
```
Comment (wboereboom) [inline src/main/java/com/adyen/model/terminal/AmountsResp.java:110]: This type seems fishy. isn't amount always an integer or long?

File: src/main/java/com/adyen/terminal/TerminalAPIRequest.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
+/*
```
Comment (wboereboom) [inline src/main/java/com/adyen/terminal/TerminalAPIRequest.java:1]: let's get rid of these headers now

Comment (jillingk) [reply src/main/java/com/adyen/terminal/TerminalAPIRequest.java]: done

File: src/main/java/com/adyen/model/terminal/AmountsResp.java (hunk 2 above)
Comment (jillingk) [reply src/main/java/com/adyen/model/terminal/AmountsResp.java:110]: Actually we already used this type in the old nexo models, let me double check tho

File: README.md [hunk 4]
```diff
@@ -284,7 +284,7 @@ In order to submit In-Person requests with [Terminal API over Cloud](https://doc
 import com.adyen.Client;
 import com.adyen.enums.Environment;
 import com.adyen.service.TerminalCloudAPI;
-import com.adyen.model.nexo.*;
+import com.adyen.terminal.*;
```
Comment (wboereboom) [inline README.md:287]: Should we make it more clear what we should import from where? I think the `com.adyen.terminal.*` import is pretty fuzzy compared to how other services work

Comment (jillingk) [reply README.md:298]: Updated the imports in the readme, should we proceed with this?

Comment (jillingk) [reply README.md]: Added implicit imports

**Review by wboereboom — APPROVED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**